### PR TITLE
Central London

### PR DESCRIPTION
**Pull request by doismellburning**

### CODE REVIEW

**Comment by ntlk** [comment]
I've already merged but only just notices that the Central London page still has the info about Pembury and incorrect date?

File: uk_central_london.html [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+layout: base
+title: Computer Anonymous Central London
+---
+<h1>Computer Anonymous — Central London</h1>
+<p>
+    Read <a href="index.html">The overview</a> if you haven't already, especially the code of conduct.
+</p>
+
+<h2>The Meetings</h2>
+      <p>We meet on the first Thursday of the month, at <a href="http://www.individualpubs.co.uk/pembury/">The Pembury Tavern</a>, <a href="http://osm.org/go/euu7IkpEN-?m=">90 Amhurst Rd, E8 1JH</a></p>
```
Comment (timcowlishaw) [inline uk_central_london.html:11]: Looks like there's still a reference to the Pembury in here!

**Comment by gearmonkey** [comment]
fixed in #170

**Comment by timcowlishaw** [comment]
Hah, also fixed in #171. race conditions ftw.

**Pull request merged**